pkg/api/api.go

```diff
@@ -419,7 +419,7 @@ func (hs *HTTPServer) registerRoutes() {
 			annotationsRoute.Put("/:annotationId", routing.Wrap(UpdateAnnotation))
 			annotationsRoute.Patch("/:annotationId", routing.Wrap(PatchAnnotation))
 			annotationsRoute.Post("/graphite", reqEditorRole, routing.Wrap(PostGraphiteAnnotation))
-			annotationsRoute.Get("/tags", routing.Wrap(GetAnnotationTags))
+			annotationsRoute.Get("/tags", authorize(reqSignedIn, ac.EvalPermission(ac.ActionAnnotationsTagsRead)), routing.Wrap(GetAnnotationTags))
```

Comment:
nit: I think we might want to have scope here as well, based on tag id.

Reply:
Added Scope `ScopeAnnotationsTagsAll` to this endpoint